src/ctap/mod.rs

```diff
@@ -109,6 +110,10 @@ pub const ES256_CRED_PARAM: PublicKeyCredentialParameter = PublicKeyCredentialPa
     cred_type: PublicKeyCredentialType::PublicKey,
     alg: SignatureAlgorithm::ES256,
 };
+// You can change this value to one of the following for more privacy.
+// - Some(CredentialProtectionPolicy::UserVerificationOptionalWithCredentialIdList)
+// - Some(CredentialProtectionPolicy::UserVerificationRequired)
+const DEFAULT_CRED_PROTECT: Option<CredentialProtectionPolicy> = None;
```

Comment:
Unrelated to this PR but I think we need to move these configuration constants from compile-time to one-time-programmable area in the storage.

Reply:
So if I manually update, I don't override my custom settings? Sounds good.

Reply:
Note that with the current storage, "persistent" entries can be implemented by not deleting them when calling [reset](https://github.com/google/OpenSK/blob/af79d92a888fb398bd1e630f1d821c4ed4128f95/src/ctap/storage.rs#L397) (which currently deletes all entries). With the next storage, "persistent" entries would simply have a key below the persistent/reset threshold (and the reset operation would be atomic, once started it cannot be stopped).

Note also that the test suite might need to support the fact that the persistent storage don't get fully reset when reset is called (if those persistent entries may alter the test suite behavior). We may provide an additional custom CTAP command to fully reset the storage (including persistent entries).

If we don't want to wait the next storage, I can implement a facility to simplify adding "persistent" (that survive reset) entries in the current one (so that we only need to modify a predicate on entries whether they are persistent). Tell me.

Reply:
No worries, I think we can worry about our compile-time constants later.